regression/test.pl

```diff
@@ -289,6 +301,8 @@ ($$$$)
              as runs with different suffixes will operate independently and keep
              independent logs.
   -f         forward the test name to CMD
+  -S <0|1>   (un-)suppress color output, default 0 if TESTPL_SUPPRESS_COLOR_OUTPUT
```

Comment:
Rather than -S 0/1 to turn this on/off, could you have say "-P" for "plain output" - and just default to colour unless "-P" or TESTPL_COLOR_OUTPUT=0 ?

Reply:
@chrisr-diffblue I wanted a way to force it back on through command line arguments if `TESTPL_SUPPRESS_COLOR_OUTPUT=1`